src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -1487,6 +1487,7 @@ internal ImmutableArray<BoundExpression> MakeTempsForDiscardArguments(ImmutableA
             return arguments;
         }
 
+#nullable disable
```

Comment:
Too much churn for this PR? If so, please add a prototype to re-enable annotations, as this is all "new" code that we want to add to the codebase.

Reply:
Added a PROTOTYPE comment 👍